pycbc/workflow/jobsetup.py

```diff
@@ -1225,7 +1488,7 @@ def create_node(self, bank):
             curr_tag = 'bank%d' %(i)
             # FIXME: What should the tags actually be? The job.tags values are
             #        currently ignored.
-            curr_tags = bank.tags + [curr_tag]
+            curr_tags = bank.tags + [curr_tag] + self.tags
```

Comment:
Have done as you suggest in https://github.com/a-r-williamson/pycbc/commit/e2910dc7c51949b3e8cc8ab28d6a2e75226c3945.